extensions/BMO/template/en/default/bug/create/create-recruiting.html.tmpl

```diff
@@ -183,6 +169,7 @@ function jobDescToggle(what) {
             <option value="Tyler Haugen">Tyler Haugen</option>
             <option value="Josh Kruse">Josh Kruse</option>
             <option value="Matt Frassica">Matt Frassica</option>
+            <option value="Matt Frassica">Huma Shah</option>
```

Comment:
<option value="Matt Frassica">Huma Shah</option>
 should be
<option value="Huma Shah">Huma Shah</option>